portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyBlogTests.testcase

```diff
@@ -0,0 +1,66 @@
+@component-name = "portal-lima"
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the Blogs Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Blogs Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "Blogs Page",
+				widgetName = "Blogs");
+
+			Navigator.gotoPage(pageName = "Blogs Page");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Check that is possible to edit a blog"
+	@priority = 5
+	test CanEditBlog {
+		task ("And a Blog Entry exists") {
+			JSONBlog.addEntry(
+				entryContent = "Blog Entry Content",
+				entryTitle = "Blog Entry Title");
+
+			Refresh();
```

Comment:
You could also do as before, using the `Blogs.addEntry` macro, but your solution is also correct! You don't have to change anything, thank you!!